introduction.md

```diff
@@ -3,3 +3,8 @@ Hello,
 My name is Anders, and I'm working on the dojo_rules. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?